cmd/mo-service/main.go

```diff
@@ -71,14 +71,71 @@ func waitSignalToStop(stopper *stopper.Stopper) {
 func startService(cfg *Config, stopper *stopper.Stopper) error {
 	// TODO: start other service
 	switch strings.ToUpper(cfg.ServiceType) {
+
 	case cnServiceType:
 		return startCNService(cfg, stopper)
+
 	case dnServiceType:
 		return startDNService(cfg, stopper)
+
 	case logServiceType:
-		return startLogService(cfg, stopper)
+		_, err := startLogService(cfg, stopper)
+		if err != nil {
+			return err
+		}
+		return nil
```

Comment:
what's problem? Why not use "return startLogService(cfg, stopper)"

Reply:
startLogService returns two values.

Reply:
waiting for bootstrap is not needed, removed.